lib/Config.cpp

```diff
@@ -45,6 +45,13 @@ void Albinos::Config::parseResponse(json const &data)
     return;
   } catch (...) {
   }
+
+  try {
+    settingNames.reset(new std::vector<std::string>(std::move(data.at("SETTINGS_NAME").get<std::vector<std::string>>())));
```

Comment:
le bon field c'est SETTINGS_NAMES il me semble, si tu cherche a recupe le resultat d'un get settings names

Reply:
```suggestion
    settingNames.reset(new std::vector<std::string>(std::move(data.at("SETTINGS_NAMES").get<std::vector<std::string>>())));
```

Reply:
Ah, that's not proper English so I got tricked :P

Reply:
its not ?

Reply:
I don't know how to explain, but I guess "setting name" is a compound noun and therefor it's pluralised to "setting names".